resources/test/tests/functional/api-tests-1.html

```diff
@@ -343,7 +343,7 @@ <h1>Sample HTML5 API Tests</h1>
     {
       "status_string": "FAIL",
       "name": "Test throw SyntaxError DOMException where JS SyntaxError expected; expected to fail",
-      "message": "assert_throws_js: function \"function () {document.querySelector(\"\")}\" threw object \"SyntaxError: '' is not a valid selector\" (\"SyntaxError\") expected instance of function \"function SyntaxError() {\n    [native code]\n}\" (\"SyntaxError\")",
+      "message": "assert_throws_js: function \"function () {document.querySelector(\"\")}\" threw object \"SyntaxError: Document.querySelector: '' is not a valid selector\" (\"SyntaxError\") expected instance of function \"function SyntaxError() {\n    [native code]\n}\" (\"SyntaxError\")",
```

Comment:
We can't be testing the error message, right?

Reply:
We totally are.  :(  See https://github.com/web-platform-tests/wpt/pull/21762#issuecomment-585425959 and https://github.com/web-platform-tests/wpt/pull/21762#issuecomment-585426827

Reply:
I see, this is specifically for testharness.js.

Reply:
Yeah, this is the "does testharness behave right?" test.  We could change testharness to not include the message in its reporting, but that would make its actual failure reporting less useful... We could also change this test to not check for message identity, of course...